omod/src/main/java/org/openmrs/module/registrationapp/RegistrationAppUiUtils.java

```diff
@@ -157,20 +157,16 @@ public static void validateLatitudeAndLongitudeIfNecessary(PersonAddress address
 			
 	        Map<String, String> regex = Context.getRegisteredComponent(AddressSupportCompatibility.ID, AddressSupportCompatibility.class).getElementRegex();
 
-			if (StringUtils.isNotBlank(address.getLatitude())) {
-				if (regex != null && StringUtils.isBlank(regex.get("latitude"))) {
-					if (!RegistrationAppUiUtils.isValidLatitude(address.getLatitude())) {
+			if (StringUtils.isNotBlank(address.getLatitude()) && regex != null
+			        && StringUtils.isBlank(regex.get("latitude"))
+			        && !RegistrationAppUiUtils.isValidLatitude(address.getLatitude())) {
 						errors.reject("registrationapp.latitude.invalid");
-					}
-				}
 			}
 			
-			if (StringUtils.isNotBlank(address.getLongitude())) {
-				if (regex != null && StringUtils.isBlank(regex.get("longitude"))) {
-					if (!RegistrationAppUiUtils.isValidLongitude(address.getLongitude())) {
+			if (StringUtils.isNotBlank(address.getLongitude()) && regex != null
+			        && StringUtils.isBlank(regex.get("longitude"))
+			        && !RegistrationAppUiUtils.isValidLongitude(address.getLongitude())) {
 						errors.reject("registrationapp.longitude.invalid");
```

Comment:
```suggestion
				&& StringUtils.isBlank(regex.get("longitude"))
				&& !RegistrationAppUiUtils.isValidLongitude(address.getLongitude())) {
					errors.reject("registrationapp.longitude.invalid");
```
Spaces are bad, tabs are good. Additionally, you have one tab too many in one place.

Reply:
Am resolving this soon